pyrokinetics/gk_code/GKInputCGYRO.py

```diff
@@ -491,3 +577,35 @@ def set(
             return
 
         self.data = convert_dict(self.data, local_norm.cgyro)
+
+    def _check_basic_miller(self):
+        """
+        Checks if CGYRO input file is a basic Miller geometry by seeing if moments that are higher than triangularity
+        are 0
+        Returns
+        -------
+        is_basic_miller: Boolean
+            True if Miller, False is MXH
+        """
+
+        mxh_only_parameters = [
+            "ZETA",
+            "S_ZETA",
+            "SHAPE_COS0",
+            "SHAPE_COS1",
+            "SHAPE_COS2",
+            "SHAPE_COS3",
+            "SHAPE_SIN3",
+            "SHAPE_S_COS0",
+            "SHAPE_S_COS1",
+            "SHAPE_S_COS2",
+            "SHAPE_S_COS3",
+            "SHAPE_S_SIN3",
+        ]
+
+        is_basic_miller = True
+        for param in mxh_only_parameters:
+            if self.data.get(param, 0.0) != 0:
+                is_basic_miller = False
```

Comment:
A very minor point: Once `is_basic_miller` is set to false, there's no need to continue in the for loop so we may as well `break`, or instead could just `return false`.